internal/xds/translator/translator.go

```diff
@@ -52,6 +52,10 @@ func Translate(ir *ir.Xds) (*types.ResourceVersionTable, error) {
 			Domains: httpListener.Hostnames,
 		}
 
+		if len(vHost.Domains) == 0 {
```

Comment:
afaik this shouldnt be needed here, the Gateway API translator should have set this to `*` while creating the IR
https://github.com/envoyproxy/gateway/blob/a4773ec06b675cc4a6ee1220d0c8ece09053471c/internal/ir/xds.go#L84

Reply:
looks like `Validate` is not called today, will raise a PR once https://github.com/envoyproxy/gateway/pull/216 is in to call Validate before performing a `Store`

Reply:
raised https://github.com/envoyproxy/gateway/pull/276